core/templates/pages/exploration-editor-page/translation-tab/state-translation/state-translation.component.html

```diff
@@ -248,7 +248,7 @@ <h3 tabindex="0">Content</h3>
                                      [isResponse]="true"
                                      [showWarning]="contentIdNeedUpdates(answerGroup.outcome.feedback.contentId)"
                                      (navigateToState)="navigateToState($event)"
-                                     [correctnessFeedbackEnabled]="explorationCorrectnessFeedbackService.isEnabled()">
+                                     [correctnessFeedbackEnabled]="true">
```

Comment:
Why keep this? We need to get rid of correctnessFeedbackEnabled entirely from the codebase, with very few exceptions as discussed in code review. Otherwise we're just carrying around dead code with us.

Reply:
My bad this is totally unnecessary. I will remove it.

Reply:
Done.